src/utils/Config.java

```diff
@@ -564,6 +577,7 @@ protected void setDefaults() {
     default_map.put("tsd.core.stats_with_port", "false");    
     default_map.put("tsd.http.show_stack_trace", "true");
     default_map.put("tsd.http.query.allow_delete", "false");
+    default_map.put("tsd.http.header_tag", "");
```

Comment:
The "header_tag" config parameter is actually a string, representing the name of the HTTP header to look for.
ie `tsd.http.header_tag  = X-TSDB-TAG`

Do you suggest to add a header_tag_enable boolean config parameter to enable/disable this behaviour ?